test/testgroups/opsrccsctests.go

```diff
@@ -50,12 +50,12 @@ func OpSrcCscTestGroup(t *testing.T) {
 	require.NoError(t, err, "CatalogSourceConfig child resources were not created")
 
 	// Run the test suites.
-	if proxyAPIAvailable {
+	if proxyTestsEnabled {
 		t.Run("proxy-test-suites", testsuites.ProxyTests)
 	}
-	t.Run("opsrc-creation-test-suite", testsuites.OpSrcCreation)
-	t.Run("csc-target-namespace-test-suite", testsuites.CscTargetNamespace)
-	t.Run("packages-test-suite", testsuites.PackageTests)
-	t.Run("csc-invalid-tests", testsuites.CscInvalid)
-	t.Run("watch-tests", testsuites.WatchTests)
+	// t.Run("opsrc-creation-test-suite", testsuites.OpSrcCreation)
```

Comment:
I am assuming you missed reverting this before pushing :-)

Reply:
That is correct.